.github/workflows/wheels.yml

```diff
@@ -152,37 +101,48 @@ jobs:
         with:
           platforms: aarch64
 
-      ## Build
+      # Get sdist
+      - uses: actions/download-artifact@v3
+        with:
+          name: artifact
+          path: dist
 
-      - uses: pypa/cibuildwheel@v2.4.0
+      ## Build
+      - uses: pypa/cibuildwheel@v2.6.0
+        with:
+          package-dir: dist/h3-*.tar.gz
+          output-dir: dist
         env:
           CIBW_TEST_REQUIRES: pytest numpy
-          CIBW_TEST_COMMAND: pytest {project}/tests
+          CIBW_TEST_COMMAND: pytest h3-py/tests
           CIBW_ARCHS_LINUX: auto aarch64
           CIBW_ARCHS_MACOS: x86_64 arm64
           CIBW_BUILD: ${{ matrix.build }}
           CIBW_SKIP: ${{ matrix.skip }}
 
-      - name: Check with Twine
-        run: |
-          pipx run twine check wheelhouse/*
-
       - name: Upload artifacts to GitHub
         uses: actions/upload-artifact@v3
         with:
-          path: wheelhouse/*.whl
+          path: dist/*.whl
 
   upload_all:
-    needs: [make_sdist, make_cibw_v1_wheels, make_cibw_v2_wheels]
+    needs: [make_sdist, make_cibw_v2_wheels]
     runs-on: ubuntu-latest
     if: github.event_name == 'release' && github.event.action == 'published'
 
     steps:
+      - name: Setup Python
+        uses: actions/setup-python@v3
+
       - uses: actions/download-artifact@v3
         with:
           name: artifact
           path: dist
 
+      - name: Check with Twine
+        run: |
+          pipx run twine check dist/*
```

Comment:
Is `pipx` installed with `actions/setup-python`? Do you need to `pip install twine`, or does `pipx run` do that automatically if not installed?

Reply:
These are good questions! I haven't entirely figured out what exactly `actions/setup-python` does and doesn't do. If you have any idea, that'd be a big help. Otherwise, I'll try to figure this out eventually.